twilio/twiml/voice_response.py

```diff
@@ -135,6 +137,13 @@ def gather(self,
             timeout=timeout,
             finish_on_key=finish_on_key,
             num_digits=num_digits,
+            partial_result_callback=partial_result_callback,
+            partial_result_callback_method=partial_result_callback_method,
+            language=format_language(language),
+            hints=hints,
+            barge_in=barge_in,
+            acknowledge_sound_url=acknowledge_sound_url,
+            input=input,
```

Comment:
They did.  While it's a built-in function, it's not reserved like "from".  One could argue that there's potential that overwriting the value of input in this scope could cause errors; but, I seriously doubt it would.  Happy to change it to "input_" if that's a preferred strategy.